cmd/cspc-operator/app/handler.go

```diff
@@ -95,6 +94,13 @@ func (c *Controller) syncHandler(key string) error {
 	// TODO: Deep-copy only when needed.
 	cspcGot := cspc.DeepCopy()
 	err = c.syncCSPC(cspcGot)
+	if err != nil {
+		return err
+	}
+	//cleaning up CSPI resources in case of CSPC scale down or CSPI deletion
```

Comment:
instead of saying 'CSPC scale down', better to put it as 'removing poolSpec from CSPC'
what is the case regarding 'CSPI deletion'?

Reply:
CSPI deletion is the manual deletion of CSPI.